src/main/java/org/kainos/ea/api/JobService.java

```diff
@@ -12,13 +15,12 @@
 
 
 public class JobService {
-    private JobDao jobDao;
+    private JobDao jobDao = new JobDao();
```

Comment:
This will be overwritten by anything passed into the constructor, no need to instantiate it.